using-valtimo/form-flow/create-form-flow-definition.md

```diff
@@ -203,7 +203,7 @@ available can be found [here](/reference/modules/form-flow.md#available-properti
                     "definition": "end-form"
                 }
             },
-            "onComplete": ["${someDocumentService.saveFormFlowResult(additionalProperties, step.submissionData)}"]
+            "onComplete": ["valtimoFormFlow.completeTask(additionalProperties, step.submissionData, {'doc:/':''})"]
```

Comment:
is this change correct? it does not match the code in the test implementation.

Reply:
It is correct. But I removed it, since it doesn't make the example more clear.